prow/gerrit/client/client.go

```diff
@@ -109,6 +99,19 @@ func (p ProjectsFlag) Set(value string) error {
 	return nil
 }
 
+// ProjectsFlagToConfig converts Gerrit project configured as command line args
+// to prow config struct for backward compatilibity
+func ProjectsFlagToConfig(hostProjects ProjectsFlag) map[string]map[string]*config.GerritQueryFilter {
```

Comment:
Nit: There might be fewer changes if we just updated the underlying type of `ProjectsFlag` to be the new type. It doesn't really matter though, this works too.